dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -664,28 +752,61 @@ protected void updateItem(final Ds3TreeTableValue.Type item, final boolean empty
         });
     }
 
+
+    private void manageItemsCount(final TreeItem<Ds3TreeTableValue> selectedItem) {
+        if (ds3TreeTable.getSelectionModel().getSelectedItems().size() == 1
+                && selectedItem.getValue().getType().equals(Ds3TreeTableValue.Type.File)) {
+            ds3Common.getDs3PanelPresenter().getInfoLabel().setVisible(false);
+            ds3Common.getDs3PanelPresenter().getCapacityLabel().setVisible(false);
+        } else {
+            ds3Common.getDs3PanelPresenter().getInfoLabel().setVisible(true);
+            ds3Common.getDs3PanelPresenter().getCapacityLabel().setVisible(true);
+            ds3Common.getDs3PanelPresenter().getInfoLabel().setText("Calculating.......");
+            ds3Common.getDs3PanelPresenter().getCapacityLabel().setText(resourceBundle.getString("infoLabel"));
+            ds3PanelPresenter.calculateFiles(ds3TreeTable);
+        }
+    }
+
     private void showPhysicalPlacement() {
-        final ImmutableList<TreeItem<Ds3TreeTableValue>> values = ds3TreeTable.getSelectionModel().getSelectedItems()
+        ImmutableList<TreeItem<Ds3TreeTableValue>> tempValues = ds3TreeTable.getSelectionModel().getSelectedItems()
                 .stream().collect(GuavaCollectors.immutableList());
-        if (values.isEmpty()) {
+        final TreeItem<Ds3TreeTableValue> root = ds3TreeTable.getRoot();
+        if (tempValues.isEmpty()) {
             LOG.error("Nothing selected");
             ALERT.setContentText("Nothing selected !!");
             ALERT.showAndWait();
             return;
+        } else if (tempValues.isEmpty()) {
+            final ImmutableList.Builder<TreeItem<Ds3TreeTableValue>> builder = ImmutableList.builder();
+            tempValues = builder.add(root).build().asList();
```

Comment:
If this is just adding a single value this can be simplified to:

```java

tempValues = ImmutableList.of(root);
```

I'm also not sure that the `asList()` call is necessary.